projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/intro/thruul.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "大师熔炼术",
+  "text": [
+    {
+      "text": "看来你已经进步到这里了工匠，也许你比我想的更聪明。是时候让你的初级熔化炉和那些初级金属变得更强大了！"
```

```suggestion
      "text": "看来你已经进步到这里了，工匠。也许你比我想的更聪明。是时候让你的低级熔化炉和那些低级金属变得更强大了！"
```
这句话感觉不太对劲。持保留意见。“熔化炉”应当指的是小的“焦黑熔化炉”？